### PR TITLE
Feature/comments

### PR DESCRIPTION
**Pull request by FiloSanza**
Add endpoint to create comment + comment factory.

View components to show the comments will be added later when post-group-user integration will be completed in order to be able to test everything together.

### CODE REVIEW

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -34,51 +14,18 @@ public function create()
...
-     * Remove the specified resource from storage.
-     *
-     * @param  int  $id
-     * @return \Illuminate\Http\Response
-     */
-    public function destroy($id)
-    {
-        //
+        //TODO redirect to post
+        return Response('Success', 200);
```
Comment (FiloSanza) [inline app/Http/Controllers/CommentController.php:29]:
```suggestion

        return redirect()->route('posts.show', $request->post_id);
```

**Review by FiloSanza — COMMENTED** [review]
remove todo

File: database/factories/CommentFactory.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    /**
+     * Define the model's default state.
+     *
+     * @return array<string, mixed>
+     */
+    public function definition()
+    {
+        return [
+            // TODO Post::factory() per post_id
+            'post_id' => NULL,
```
Comment (FiloSanza) [inline database/factories/CommentFactory.php:28]:
```suggestion
            'post_id' => post::Factory,
```
```suggestion
            'post_id' => Post::factory(),
```

**Review by LorenzoDrudi — APPROVED** [review]
lgtm

**Pull request merged**